internal/ethapi/api.go

```diff
@@ -609,6 +609,18 @@ func (s *PublicBlockChainAPI) ChainId() (*hexutil.Big, error) {
 	return nil, fmt.Errorf("chain not synced beyond EIP-155 replay-protection fork block")
 }
 
+func (s *PublicBlockChainAPI) GetUpgrades(ctx context.Context) *params.UpgradeConfig {
```

Comment:
Can we simplify this functionality and just add an API to return the entire `ChainConfig` so that it includes everything and we only add one API to support?

Reply:
Makes sense 👍

Reply:
done